plugins/modules/route53.py

```diff
@@ -604,6 +659,30 @@ def main():
         'HealthCheckId': health_check_in,
         'SetIdentifier': identifier_in,
     })
+
+    if geo_location:
+        continent_code = geo_location.get('continent_code')
+        country_code = geo_location.get('country_code')
+        subdivision_code = geo_location.get('subdivision_code')
+
+        if continent_code and (country_code or subdivision_code):
+            module.fail_json(changed=False, msg='While using geo_location, continent_code is mutually exclusive with country_code and subdivision_code.')
+
+        if not continent_code and not country_code and not subdivision_code:
```

Comment:
Something like this should do the job.
```suggestion
        if not all([continent_code, country_code, subdivision_code]):
```

Reply:
Fixed.